WooCommerce/StoreWidgets/Entitlements/StoreWidgetsExtensionRelease-Alpha.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.com.automattic.woocommerce.alpha</string>
+	</array>
+</dict>
+</plist>
```

Comment:
One could argue we ought to use the same file for Debug and Release... The current setup matches what the app target does, but I could be persuaded to make the change 😄

Reply:
Maybe it'll be good to match the file names too, like `StoreWidgetsExtension-(Alpha|Debug|Release)`?

Reply:
Hey @crazytonyli. Sorry I saw this comment only after merging already...

Followed up in https://github.com/woocommerce/woocommerce-ios/pull/7674

Reply:
No problem. Thanks for following up!